Curricular Models/GenEvo/GenEvo 4 Competition.nlogo

```diff
@@ -389,9 +389,9 @@ Because the molecular interactions in each cell model are stochastic, the popula
 
 You can use this model in two ways.
 
-1. The simplest way to use this model is to set CHOOSE-MODELS? to OFF. In this mode, the default Genetic Switch model is loaded for each cell. This method simulates genetic drift as a mechanism of evolution since each cell starts off identical.
+1. The simplest way to use this model is to set CHOOSE-MODELS? to OFF. In this mode, the default Genetic Switch model is created for each cell. This method simulates genetic drift as a mechanism of evolution since each cell starts off identical.
 
-2. If CHOOSE-MODELS? is ON, a user can manually select the NetLogo models (`.nlogo` files) to be loaded for each cell, allowing each cell to have a different genetic switch. This allows for variability in the population. This method simulates natural selection as well as genetic drift.
+2. If CHOOSE-MODELS? is ON, a user can manually select the NetLogo models (`.nlogo` files) to be created for each cell, allowing each cell to have a different genetic switch. This allows for variability in the population. This method simulates natural selection as well as genetic drift.
```

Comment:
I will adjust this one too. We need to be careful to not make it sound as if it is the file that is created.